ci-operator/config/openshift/release/openshift-release-master__nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -90,6 +90,17 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-upgrade-aws-single-node
+- as: e2e-aws-upgrade-ovn-metal-single-node-rt
```

Comment:
This is going to cause problems for how we report what this job is.  If it's metal, the prior art is to just say metal and not mention the underlying platform provider.

That lets us attribute the job results to the right place.

Suggest: e2e-metal-ovn-single-node-rt-upgrade